cmd/pbm/main.go

```diff
@@ -33,6 +33,7 @@ var (
 	configShowKey       = configCmd.Arg("key", "Show the value of a specified key").String()
 
 	backupCmd      = pbmCmd.Command("backup", "Make backup")
+    bcpAdminTag    = backupCmd.Flag("tagname", "Set Tag for preferred Secondary <name=value>").StringMap()
```

Comment:
But it's more desirable that this be a part of the config. Eg.

```yaml
backup:
  replset_tag_filter:  #children below here can be arbitrary key-value pairs. But no grandchildren.
     dc: east
     usage: reporting
```

Reply:
I thought it would be better specify by pbm cli the tag via single key-value pair. But your proposed solution maybe it's way better than mine. I will try to implement this feature. Thanks for your suggest